sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Interface/IBlobService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+namespace Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.Interface
+{
+    internal interface IBlobService
+    {
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="uri"></param>
+        /// <param name="buffer"></param>
+        /// <param name="fileRelativePath"></param>
+        /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
+        Task UploadBufferAsync(string uri, string buffer, string fileRelativePath);
+         string GetCloudFilePath(string uri, string fileRelativePath);
+        void UploadBlobFile(string uri, string fileRelativePath, string filePath);
```

Comment:
pls add test for this as well

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/46807#discussion_r1822300555